code-scanning/trivy-iac-scan.yml

```diff
@@ -29,17 +29,15 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Build an image from Dockerfile
-        run: |
-          docker build -t docker.io/my-organization/my-app:${{ github.sha }} .
-
-      - name: Run Trivy vulnerability scanner
-        uses: aquasecurity/trivy-action@7b7aa264d83dc58691451798b4d117d53d21edfe
+      - name: Run Trivy vulnerability scanner in IaC mode
+        uses: aquasecurity/trivy-action@0.20.0
```

Comment:
There have been 4 minor releases since this PR was authored...
```suggestion
        uses: aquasecurity/trivy-action@0.24.0
```

(Note that the `runs-on` field probably should be bumped for this file as well... but that line isn't within reach of comments atm)